mobile-app/lib/providers/account_associations_providers.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:quantus_sdk/quantus_sdk.dart';
+
+class AccountAssociationsNotifier extends StateNotifier<AsyncValue<AccountAssociations>> {
+  final TaskmasterService _taskmasterService = TaskmasterService();
+
+  AccountAssociationsNotifier() : super(const AsyncValue.loading()) {
+    fetchAssociations();
+  }
+
+  Future<void> fetchAssociations() async {
+    try {
+      final associations = await _taskmasterService.getAccountAssociations();
+
+      state = AsyncValue.data(associations);
+    } catch (e, st) {
+      state = AsyncValue.error(e, st);
```

Comment:
Please put an error printout any time there's an error. As a matter of principle. 

Even though here the error is hopefully handled elsewhere we should not have silent catch statements.